cypress/integration/sales/void_a_sales_invoice_then_invoice_the_billing_candidates_again.js

```diff
@@ -29,21 +29,49 @@ import { getLanguageSpecific } from '../../support/utils/utils';
 import { salesInvoices } from '../../page_objects/sales_invoices';
 
 describe('Void Sales Invoice and invoice the billing candidates again', function() {
-  const businessPartnerName = 'Test Lieferant 1';
-  const productName = 'Convenience Salat 250g';
-  const originalQuantity = 200;
-  const shipmentNotificationInboxText = `Lieferung [number] für Partner ${businessPartnerName} wurde erstellt.`;
-  const shipmentNotificationModalText = 'Created: 1 WorkPackage Queue;';
-  const shipmentQuantityTypeOption = 'Quantity to deliver';
-  const generateInvoicesNotificationModalText =
-    'Fakturlauf mit 1 Rechnungen eingeplant. Es sind bereits 0 zu erstellende Rechnungen in der Warteschlange, die vorher verarbeitet werden.';
-
-  const salesInvoiceDocumentType = 'Sales Invoice';
+  // const businessPartnerName = 'Test Lieferant 1';
```

Comment:
Can the comments here be deleted please?

Reply:
yes, done.